src/App.jsx

```diff
@@ -1,9 +1,71 @@
-import './App.css';
+import "./App.css";
+import { useState, useEffect } from "react";
+import { Route, Routes, Link } from "react-router-dom";
+import ContactList from "./components/ContactList";
+import ContactProfile from "./components/ContactProfile";
+import CreateContact from "./components/CreateContact";
 
-function App() {
-    return (
-        <p>Hello, world!</p>
+export default function App() {
+  const [contacts, setContacts] = useState([]);
```

Comment:
probably best to have this inside the contacts list and fetch it there

when you get to the create page -> no need for state, you just send a POST request

when you get to the view contact by id -> you just fetch that contact when you load that component

up to you, it's ok if you keep as is right now